main.cpp

```diff
@@ -79,43 +87,84 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
+bool allowedToDrinkAlcohol (bool adult, bool insideBar, bool openHours)
+{
+    ignoreUnused (adult, insideBar, openHours);
+    return {};
+}
 
 /*
  2)
  */
-
+bool boilWater (int waterTemperature, int boilingTemperature = 100)
+{
+    ignoreUnused (waterTemperature, boilingTemperature);
+    return {};
+}
 /*
  3)
  */
-
+int parcelCounterDeliver (int john, int max, int gabriel)
+{
+    ignoreUnused (john, max, gabriel);
+    return {};
+}
 /*
  4)
  */
-
+int buildingNeighbours (int floor0, int floor1, int floor2, int floor3 = 4)
+{
+    ignoreUnused (floor0, floor1, floor2, floor3);
+    return {};
+}
 /*
  5)
  */
-
+char secondLetterInAWord (string word, int position = 2)
+{
+    ignoreUnused (word, position);
+    return {};
+}
 /*
  6)
  */
-
+char studentGrade (double exercises, double behaviour, int attendance)
+{
+    ignoreUnused (exercises, behaviour, attendance);
+    return {};
+}
 /*
  7)
  */
-
+float countryGDPInTrillions (float privateConsumption, float grossPrivateInvestment, float governmentInvestment, float governmentSpending, float exports, float imports)
```

Comment:
- [ ] revise, pick a function name that represents an action and contains a verb